src/rhsm/connection.py

```diff
@@ -533,7 +510,7 @@ def _request(self, request_type, method, info=None):
         if not len(result['content']):
             return None
 
-        return json.loads(result['content'], object_hook=self._decode_dict)
+        return json.loads(result['content'])
```

Comment:
Do we not still need `_decode_dict` and `_decode_list` in Python 2 to get everything converted to UTF8 or is that being handled elsewhere?